src/main.tsx

```diff
@@ -11,15 +11,22 @@ import { CompletedRun, EMS, LiveRun } from "components/EMS";
 import Navbar from "components/Navbar";
 import NotFound from "components/shared/NotFound";
 
+const GRAPHQL_WS_ENDPOINT =
+  import.meta.env.VITE_GRAPHQL_WS_ENDPOINT || "ws://localhost:8000/graphql/";
+
+const GRAPHQL_HTTP_ENDPOINT =
+  import.meta.env.VITE_GRAPHQL_HTTP_ENDPOINT ||
+  "http://localhost:8000/graphql/";
+
```

Comment:
I think we should modify this a little bit to give developers the option of pointing to either their local BE server or to heroku. That way, we can do FE development without ever having to spin up a local BE server.

We can do this by adding an .env.example file to git. That way, each developer can individually configure their own .env file without committing it to git.

https://vitejs.dev/guide/env-and-mode.html#env-files

Reply:
This is exactly what I do on the AWS server. The `deploy.sh` script creates a `.env.local` file with environment variables `VITE_GRAPHQL_HTTP_ENDPOINT` and `VITE_GRAPHQL_WS_ENDPOINT` that point to the heroku endpoint.

The logic in the code here simply states that if no vite env variables of those names are found, then it defaults to the localhost ports

I will update the readme

Reply:
Readme updated in fbe906c71f7d16846c6b5f2b17e49fdcd2bf1b25

Reply:
Added `.env.example` file in 729aabf04c978991072d82b82191ad9422c1f5c5